test/client/query-subscribe.js

```diff
@@ -391,18 +391,24 @@ describe('client query subscribe', function() {
   });
 
   it('changing a sorted property moves in a subscribed query', function(done) {
+    if (!makeSortedQuery) return this.skip();
+
     var connection = this.backend.connect();
+
     async.parallel([
       function(cb) { connection.get('dogs', 'fido').create({age: 3}, cb); },
       function(cb) { connection.get('dogs', 'spot').create({age: 5}, cb); }
     ], function(err) {
       if (err) return done(err);
-      var query = connection.createSubscribeQuery('dogs', {$orderby: {age: 1}}, null, function(err, results) {
-        if (err) return done(err);
-        expect(util.pluck(results, 'id')).eql(['fido', 'spot']);
-        expect(util.pluck(results, 'data')).eql([{age: 3}, {age: 5}]);
-        connection.get('dogs', 'spot').submitOp({p: ['age'], na: -3});
-      });
+      var query = connection.createSubscribeQuery(
+        'dogs', makeSortedQuery({}, [['age', 1]]), null,
```

Comment:
this is a bit busy for a single statement. Better to build object on a previous line. Annoying we are using the term `query` to mean two different things (historical nomenclature problem i've never figured out). So sometimes I use the term `expression` to mean the raw object query. You could also call it `sortedQuery` in this context or `dbQuery` if we used that consistently.

```
  var dbQuery = getSortedQuery({}, [['age', 1]]);
  ...
```